components/api_server/src/database/measurements.py

```diff
@@ -11,12 +11,21 @@
 from shared.utils.functions import iso_timestamp
 from shared.utils.type import MetricId
 
+# Projection options
+NO_ID = {"_id": False}
+NO_SOURCE_DETAILS = NO_ID | {"sources.entities": False, "sources.entity_user_data": False}
+NO_MEASUREMENT_DETAILS = NO_SOURCE_DETAILS | {"issue_status": False}
+
+# Sort optins
+START_ASCENDING = [("start", pymongo.ASCENDING), ("end", pymongo.DESCENDING)]
```

Comment:
Why does this also contain "end"?

Reply:
So that it matches the index, see first alinea here: https://www.mongodb.com/docs/manual/tutorial/sort-results-with-indexes/#std-label-sorting-with-indexes